Content.Shared/Clothing/EntitySystems/ClothingSystem.cs

```diff
@@ -140,6 +150,12 @@ protected virtual void OnGotUnequipped(EntityUid uid, ClothingComponent componen
             ToggleVisualLayer(args.Equipee, HumanoidVisualLayers.Hair, HairTag);
         if ((new string[] { "mask", "head" }).Contains(args.Slot) && _tagSystem.HasTag(args.Equipment, NoseTag))
             ToggleVisualLayer(args.Equipee, HumanoidVisualLayers.Snout, NoseTag);
+        if ((new string[] { "outerClothing" }).Contains(args.Slot) && _tagSystem.HasTag(args.Equipee, TailTag) && _tagSystem.HasTag(args.Equipment, TailTag))
+        {
+            _humanoidSystem.SetLayerVisibility(args.Equipee, HumanoidVisualLayers.Tail, true);
+            TryComp(args.Equipee, out AppearanceComponent? appearance);
+            _appearance.SetData(args.Equipee, TailVisuals.Tailcover, TailcoverVisuals.False, appearance);
+        }
```

Comment:
I'm definitely not going to ask this of you, but seeing incredibly cursed visualizer code get extended even further, makes me realize I should probably revisit this later and expand that aforementioned component into one that lets you arbitrarily declare layers to be hidden when equipping clothing with a specific condition. Harpies want to hide their tail, everyone wants to hide their hair, harpies need to hide their wings, and their tail, and their ear tufts, it's all something I could finangle into a single check.